src/CommunityToolkit.Maui.UnitTests/Converters/IntToBoolConverter_Tests.cs

```diff
@@ -44,9 +47,16 @@ public void InValidConverterValuesThrowArgumenException(object value)
 	[InlineData(2.5)]
 	[InlineData("")]
 	[InlineData(null)]
-	public void InValidConverterBackValuesThrowArgumenException(object value)
+	public void InValidConverterBackValuesThrowArgumentException(object value)
```

Comment:
Let's fix this small typo from a previous PR
```suggestion
	public void InvalidConverterBackValuesThrowArgumentException(object value)
```